src/pyosrd/agents/scheduler_agent.py

```diff
@@ -119,7 +120,7 @@ def regulated_schedule(self) -> Schedule:
     def regulate_scenario(
         self,
         scenario: str,
-        plot_all=False
+        indicator_function: Callable[[Schedule, Schedule, OSRD], pd.DataFrame]
     ) -> pd.DataFrame:
         """Regulates the given scenario using the given agent.
 
```

Comment:
Docstring shoud be updated with the up-to-date parameters

Examples would be welcome